snapper.py

```diff
@@ -484,14 +486,44 @@ def run_sync():
 
 
 def run_scrub():
-    enabled, scrub_new, check_percent, min_age = itemgetter(
-        'enabled', 'scrub_new', 'check_percent', 'min_age')(config['snapraid']['scrub'])
+    enabled, scrub_new, check_percent, min_age, scrub_delayed_run = itemgetter(
+        'enabled', 'scrub_new', 'check_percent', 'min_age', 'scrub_delayed_run'
+    )(config['snapraid']['scrub'])
 
     if not enabled:
         log.info('Scrubbing not enabled, skipping.')
 
         return None
 
+    if scrub_delayed_run and scrub_delayed_run > 0:
+        log.info('Delayed scrub is enabled.')
+
+        # get scrub_count from file or 0 if not exist or no number
+        try:
+            scrub_count = int(scrub_count_file.read_text().strip())
+        except (FileNotFoundError, ValueError):
+            scrub_count = 0
+
+        if scrub_count >= scrub_delayed_run:
+            # Run scrub job. If count is 0, scrub was forced externally
+            log.info(
```

Comment:
If you want any of this information to be pushed to Discord, make sure you use the `notifyInfo` function